core/banking-protocol/src/main/java/de/adorsys/opba/core/protocol/domain/entity/BankProfile.java

```diff
@@ -0,0 +1,38 @@
+package de.adorsys.opba.core.protocol.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.Proxy;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Proxy(lazy = false)
+public class BankProfile {
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
```

Comment:
As we have postgres strategy = GenerationType.SEQUENCE is much more performant strategy. Also it provides ids before entity is really persisted to databases when doing cascades. Please use SEQUENCE

Reply:
ok